### ISSUE TITLE
No more import button on Bandcamp and metal-archives

### ISSUE THREAD

**Issue by Tscharbo** [murdos/musicbrainz-userscripts#584]
No more import button on Bandcamp and metal-archives page!

**Comment by MCMXD** [comment]
Its not working for me either

**Comment by jeffbyrnes** [comment]
This is a duplicate of #579

**Comment by mjb187** [comment]
Even though #579 has been fixed and I've updated TaperMonkey to 5.3.2, this issue is still not fixed for me. I'm still seeing the following errors in the console:

```
@require: couldn't load @require from URL 'lib/mbimport.js': internal error
@require: couldn't load @require from URL 'lib/logger.js': internal error
@require from URL 'lib/mblinks.js': internal error
@require from URL 'lib/mbimportstyle.js': internal error
```

```
Uncaught ReferenceError: MBImportStyle is not defined
    at HTMLDocument.<anonymous> (userscript.html?name=Import-Bandcamp-releases-to-MusicBrainz.user.js&id=9930b678-2557-4a13-91ca-cd61b7cc78d9:316:5)
    at i (userscript.html?name=Import-Bandcamp-releases-to-MusicBrainz.user.js&id=9930b678-2557-4a13-91ca-cd61b7cc78d9:3:27151)
    at Object.fireWith [as resolveWith] (userscript.html?name=Import-Bandcamp-releases-to-MusicBrainz.user.js&id=9930b678-2557-4a13-91ca-cd61b7cc78d9:3:27914)
    at Function.ready (userscript.html?name=Import-Bandcamp-releases-to-MusicBrainz.user.js&id=9930b678-2557-4a13-91ca-cd61b7cc78d9:3:29707)
    at HTMLDocument.J (userscript.html?name=Import-Bandcamp-releases-to-MusicBrainz.user.js&id=9930b678-2557-4a13-91ca-cd61b7cc78d9:3:29892)
```

Any suggestions? I'm using all default settings, so if it's user error, I'm not sure how or where.

**Comment by kellnerd** [comment]
Apparently you have to reinstall the broken scripts in Tampermonkey.
I've added that hint to #579 as well.
